azure-iot-device/tests/provisioning/test_sync_provisioning_device_client.py

```diff
@@ -20,11 +23,20 @@
 fake_x509_cert_file_value = "fantastic_beasts"
 fake_x509_cert_key_file = "where_to_find_them"
 fake_pass_phrase = "alohomora"
+fake_status = "flying"
+fake_sub_status = "FlyingOnHippogriff"
+fake_operation_id = "quidditch_world_cup"
+fake_request_id = "request_1234"
+fake_device_id = "MyNimbus2000"
+fake_assigned_hub = "Dumbledore'sArmy"
+
+fake_registration_state = RegistrationState(fake_device_id, fake_assigned_hub, fake_sub_status)
 
 
 def create_success_result():
```

Comment:
Should be a fixture

Reply:
as we discussed that currently to pass fixture to the mock polling machine callback I have to also edit the logging method. also individual attribute assert can not be done , so I am keeping the function and not an fixture

---
In reply to: [315792459](https://github.com/Azure/azure-iot-sdk-python-preview/pull/169#discussion_r315792459) [](ancestors = 315792459)